src/app/views/sidebar/history/History.tsx

```diff
@@ -101,9 +102,13 @@ export class History extends Component<IHistoryProps, any> {
       element.category = date;
       items.push(element);
     });
+    return this.sortItems(items);
+  }
+
+  private sortItems = ( items: any ) => {
     items
       .sort(dynamicSort('createdAt', SortOrder.DESC))
-      .forEach((value, index) => {
+      .forEach((value : any, index: any) => {
         value.index = index;
       });
     return items;
```

Comment:
Let's use the types here instead of any especially since we know what we are sorting

Reply:
Making the changes